### DISCUSSION TITLE
Some suggestions for new labels

### DISCUSSION THREAD

**Discussion by me2beats** [godotengine/godot-proposals#4693] [General Discussions]
- `duplicate` (in both GIP and Godot repos) — to easily see that a closed issue is a duplicate
- `tracked` (in Godot main repo) to easily see that a closed issue labeled as feature proposal,  already exists in GIP.

A problem it solves is for example I want to see issues in Godot main repo labeled as `feature proposals` which are closed, but not exist in GIP and not duplicated.
Opening each issue is not handy

**Comment by Calinou** [comment]
We chose to stick to a single `archived` label for "invalid", "duplicate" and "wontfix" statuses because there aren't that many issues that are not duplicates, yet are still invalid somehow.
